### PR TITLE
Update version numbers 

### PR DESCRIPTION
**Pull request by labkey-susanh**
#### Related Pull Requests
* https://github.com/LabKey/server/pull/67

#### Changes
* Minor version updates for some of our dependencies

### CODE REVIEW

File: api/resources/credits/jars.txt [hunk 1]
```diff
@@ -19,8 +19,8 @@ fop-2.3.jar|Formatting Objects Processor (FOP)|2.3|{link:Apache|http://xmlgraphi
...
-httpclient-4.5.3.jar|Apache HTTP Client|4.5.3|{link:Apache|http://hc.apache.org/httpcomponents-client-ga}|{link:Apache 2.0|http://www.apache.org/licenses/LICENSE-2.0}|adam|HTTP client for requests of remote servers
-httpcore-4.4.6.jar|Apache HTTP Core|4.4.6|{link:Apache|http://hc.apache.org/httpcomponents-core-ga/index.html}|{link:Apache 2.0|http://www.apache.org/licenses/LICENSE-2.0}|adam|HTTP core library, HttpClient dependency
+httpclient-4.5.13.jar|Apache HTTP Client|4.5.13|{link:Apache|http://hc.apache.org/httpcomponents-client-ga}|{link:Apache 2.0|http://www.apache.org/licenses/LICENSE-2.0}|adam|HTTP client for requests of remote servers
+httpcore-4.4.14.jar|Apache HTTP Core|4.4.14|{link:Apache|http://hc.apache.org/httpcomponents-core-ga/index.html}|{link:Apache 2.0|http://www.apache.org/licenses/LICENSE-2.0}|adam|HTTP core library, HttpClient dependency
 httpmime-4.5.3.jar|Apache HTTP Mime|4.5.3|{link:Apache|http://hc.apache.org/httpcomponents-client-ga}|{link:Apache 2.0|http://www.apache.org/licenses/LICENSE-2.0}|adam|HTTP client for requests of remote servers
```
Comment (labkey-adam) [inline api/resources/credits/jars.txt:24]: Should we upgrade httpmime to 4.5.13 (latest) as well? Honestly, I can't figure out (from browsing the docs) what this thing does and whether it's a real dependency. But we thought we needed it at some point and clearly somebody is maintaining it...

Comment (labkey-adam) [reply api/resources/credits/jars.txt:24]: FYI: Assuming this upgrade goes well, we'll probably want to update httpclient, et al in the Java API.

**Review by labkey-adam — APPROVED** [review]

File: api/resources/credits/jars.txt (hunk 1 above)
Comment (labkey-susanh) [reply api/resources/credits/jars.txt:24]: We use httpmime in labkey-client-api but record that dependency here.  I'll make another pass to update more version numbers (started for labkey-client-api here: https://github.com/LabKey/labkey-api-java/tree/fb_gradle7).  These updates were just to align with updates in the gradle plugins as much as possible, for a small amount of convenience.

Comment (labkey-adam) [reply api/resources/credits/jars.txt:24]: Okay. I understand this isn't the "upgrade every JAR file" story and I'm totally fine getting existing changes merged and then circling back to jars in other places. But your call how far you want to go.

Comment (labkey-adam) [reply api/resources/credits/jars.txt:24]: > getting existing changes merged

Which I now see is what you did 😄

**Pull request merged**